index.html

```diff
@@ -1246,334 +1098,787 @@ <h3>DID URL Content Metadata</h3>
+			<li>
+			A <a>DID Resolver</a> is comparable to a web server — responsible
+			for interpreting the request and returning a representation of the
+			identified resource. Unlike general-purpose HTTP servers, DID
+			Resolvers are more constrained in their behavior, typically
+			implementing <a>DID Method</a>-specific resolution rules.
+			</li>
+			<li>
+			A client of a <a>DID Resolver</a> is comparable to a web browser —
+			responsible for constructing the DID URL and processing the
+			response. DID Resolver clients are generally more flexible than
+			browsers, applying deployment-specific handling of the resolved
+			resource by media type, including metadata and fragment (if any).
+			</li>
+		</ul>
+
+		<p>
+			The similarity ends there. DID Resolvers do not support arbitrary
+			server logic or content negotiation. Instead, the interpretation of
+			DID URL components is deterministic and governed by this
+			specification, the applicable <a>DID method</a> specification, and
+			any DID Document Properties Extensions
+			[[?DID-EXTENSIONS-PROPERTIES]] supported by the <a>DID Resolver</a>.
+		</p>
+
+		<p>
+			This section describes the structure, constraints, and required
+			handling of each DID URL component. These rules form the basis for
+			the dereferencing algorithm defined in <a
+			href="#did-url-dereferencing-algorithm">Section 5.5</a>.
+		</p>
+
+		<section id="did-url-fragment">
+			<h4>Fragment</h4>
+
+			<p>
+			A <a>DID URL</a> MAY include a
+			single fragment component, as specified in
+			<a data-cite="RFC3986#section-3.5">RFC3986 Section 3.5</a>. If
+			present, the fragment component appears after all other components
+			of the DID URL.
+			</p>
+
+			<p>
+			The fragment is not processed by the <a>DID Resolver</a>. Instead,
+			its interpretation is the responsibility of the client that receives
+			the dereferenced resource from the resolver. The meaning of the
+			fragment depends on the media type of the resolved content and may
+			be used to reference a specific node in a DID Document or a part of
+			another structured resource. For example, if the resolved resource
+			is a DID Document, the fragment may be used to identify a
+			<a data-cite="did-core#verification-methods">verification method</a>
+			to be used by the client in processing a cryptographic signature.
+			</p>
+
+			<p>
+			<a>DID Method</a>-specific and DID Document Properties Extensions
+			[[?DID-EXTENSIONS-PROPERTIES]] MUST define guidance
+			for how a <a>DID Resolver Client</a> should process a node in a <a>DID Document</a>
+			that is referenced by a fragment. Such guidance can specify
+			how to interpret the node's structure, expected data type, or its
+			use in further processing (for example, verifying a signature or
+			retrieving related resources).
+			</p>
+		</section>
+
+		<section id="did-parameters">
+			<h4>Query Parameters</h4>
+
+			<p>
+			A <a>DID URL</a> MAY include a single query
+			component, as defined in
```

Comment:
```suggestion
			A <a>DID URL</a> MAY include one or zero
			query component(s), which, if present, MAY include
			zero or more query parameters, as defined in
```

Reply:
Shouldn't this be "...if present, MAY include one or more query parameters..."